src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -391,4 +470,28 @@ protected override TokenStreamComponents CreateComponents(string fieldName, Text
             return new TokenStreamComponents(tokenizer, tok);
         }
     }
+
+    public class LuceneStartConfig
+    {
+        /// <summary>
+        /// Lucene Index Directory name
```

Comment:
is this a name or a path? Please provide an example in the comment.

Reply:
name, will add example